item_class.rb

```diff
@@ -0,0 +1,20 @@
+class Item
+  attr_accessor :id, :genre, :author, :label, :publish_date, :archived
```

Comment:
Hi @nicupop729, I verified if I put all as a reader it broke the program, the only one that could be a reader is `:publish_date`, the others need to maintain with accessor permission because for example, in your previous observation: `item. genre = self`, we are writing using the genre method to write a new value. Similar is for author, label, id, and archived.

Reply:
HI @yersel500,

Thanks a lot for reaching out!
We need to make our code clean, tidy and functional following the good practices and principles.
At the end of the day, the most important thing here is that our app is running.
Kindly, take what measures you think are needed for this to happen and ask for another review! 

You and you're team are doing great!
Keep up the good work!